libstuff/libstuff.cpp

```diff
@@ -2019,17 +2017,12 @@ bool S_sendconsume(int s, SFastBuffer& sendBuffer) {
         SINFO("Sending an ESCALATE_RESPONSE for id " << id);
     }
 
-    // Timer for tracking how long the call to send is taking to debug slow ESCALATE_RESPONSEs
-    chrono::steady_clock::time_point start = chrono::steady_clock::now();
-
     // Send as much as we can
     ssize_t numSent = send(s, sendBuffer.c_str(), sendBuffer.size(), MSG_NOSIGNAL);
     string errorMessage;
     if (numSent == -1) {
         errorMessage = " Error: "s + strerror(errno);
     }
-    SINFO("[performance] Send() took " << chrono::duration_cast<chrono::milliseconds>(chrono::steady_clock::now() - start).count()
```

Comment:
nab, this has helped diagnose many problems with buffers not emptying in the past

Reply:
It's helped with at least one for sure, but it's so specialized for specific bugs, I think we can add it back when we get to that point. Or when logrock is working, or sometime later if we need it.